tailwind.js

```diff
@@ -53,9 +53,9 @@ let colors = {
   'primary-lightest': '#D5FFF4',
   'primary-lighter': '#E6F5F0',
   'primary-light': '#69D5B1',
-  'primary-dark': '#009d6c',
+  'primary-dark': '#126D4F',
   'primary': '#00bc89',
-  'primary-darker': '#009d6c',
+  'primary-darker': '#126D4F',
```

Comment:
ha.... we have the same value. Please remove one, just make sure to remove all other places where is being used. Thanks!